.github/workflows/python.yaml

```diff
@@ -2,6 +2,8 @@ name: lakefs-spec CI
 
 on:
   push:
+    tags:
+      - "v*"
```

Comment:
As per our discussion, I would propose making this a different trigger based on release cuts on GH like so:

```yaml

on:
  workflow_dispatch:
  release:
    types:
      - published
```